Extension/src/LanguageServer/extension.ts

```diff
@@ -542,8 +540,8 @@ function onResetDatabase(): void {
 }
 
 function selectDefaultCompiler(sender?: any): void {
-    logForUIExperiment("CompilerSelect", sender);
-    clients.ActiveClient.promptSelectCompiler(true,sender);
+    logForCompilerExperiment("CompilerSelect", sender);
```

Comment:
The sender is already logged in `promptSelectCompiler`. Do we need to log another event here?

What we're actually missing is the code that queries the experiment to determine whether the new status bar button should appear or not.

Reply:
Fixed this offline with @browntarik.  Will be in the next push